.coderabbit.yaml

```
@@ -0,0 +1,72 @@
+  # Explain the reasoning behind suggestions
+  explain_reasoning: true
+
+# Integration settings
+integrations:
+  github:
+    # Auto-approve simple changes (like typos)
+    auto_approve_minor: false
+    # Request changes for critical issues
+    request_changes_critical: true
+    # Add labels to PRs
+    add_labels: true 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)